StackoverflowChatbot/Discord.cs

```diff
@@ -89,34 +93,63 @@ private static async Task ClientRecieved(SocketMessage arg)
 				}
 			}
 		}
-
```

Comment:
Please explain wtf is going on here, because the diff has totally mangled it and I can't work out what's going on.

Reply:
i just removed a blank line probably

Reply:
Okay i looked at it again, and indeed i removed a blank line, but also added a config variable so usernamelists can be accessed by the bot